pkg/crc/api/api.go

```diff
@@ -69,12 +93,20 @@ func (api CrcApiServer) handleConnection(conn net.Conn) {
 		logging.Error("Error decoding request: ", err.Error())
 		return
 	}
-
-	if handler, ok := api.handlers[req.Command]; ok {
-		result := handler(req.Args)
-		writeStringToSocket(conn, result)
+	if req.Command == "start" || req.Command == "stop" || req.Command == "delete" {
+		// queue new request to channel
```

Comment:
I believe a bit more context why they are treated differently from other requests would be useful here.
(they are slow operations, and change the VM state so they have to run sequentially. We don't want other operations querying the status of the VM to be blocked by these, so they are treated by a dedicated go routine)

Reply:
You mean in the commit log or here as a comment?

Reply:
As a comment, as for someone reading the code, it's not going to be obvious at all why we special case commands.

Reply:
Done!